frontend/common/src/helpers/poolUtils.ts

```diff
@@ -15,9 +16,13 @@ export const formattedPoolPosterTitle = ({
   stream,
   intl,
 }: formattedPoolPosterTitleProps): string => {
-  return `${title ? `${title} ` : ""}(${classification}${
-    stream ? ` ${intl.formatMessage(getPoolStream(stream))}` : ""
-  })`;
+  const streamString = stream
+    ? `${intl.formatMessage(getPoolStream(stream))}`
+    : null;
+  const genericTitle = `${classification} ${streamString}`.trim();
```

Comment:
this need a conditional for the streamString in the string literal otherwise it outputs null to screen

![image](https://user-images.githubusercontent.com/40485260/200953181-bef87358-0f77-4bc3-9a69-eb0d45c846ed.png)

Reply:
The classification can also display as null but only when run through `transformPoolToPosterTitle`.

Name, classification and stream all null on ViewPool page:
![image](https://user-images.githubusercontent.com/49166751/200968530-4710937d-25ef-483b-8a39-23b63bc02a9f.png)

On pools table: 
![image](https://user-images.githubusercontent.com/49166751/200968675-533cb767-1ed8-4e02-8a2f-388ec4038652.png)